polkadot-parachains/integritee-runtime/src/lib.rs

```diff
@@ -119,7 +119,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("integritee-parachain"),
 	impl_name: create_runtime_str!("integritee-full"),
 	authoring_version: 2,
-	spec_version: 16,
+	spec_version: 17,
```

Comment:
In this case, no bump was necessary because only state is affected. But it won't hurt, so all fine